android-components/components/support/test/src/main/java/mozilla/components/support/test/rule/Helpers.kt

```diff
@@ -37,11 +37,10 @@ private const val DEFAULT_DISPATCH_TIMEOUT_MS = 60_000L
  * @see Dispatchers.setMain
  * @see TestDispatcher
  */
-@OptIn(ExperimentalCoroutinesApi::class)
 fun runTestOnMain(
-    dispatchTimeoutMs: Long = DEFAULT_DISPATCH_TIMEOUT_MS,
+    testTimeoutMs: Duration = DEFAULT_DISPATCH_TIMEOUT_SECONDS,
     testBody: suspend TestScope.() -> Unit,
-): TestResult {
```

Comment:
I believe `TestResult` was added here probably because the result was meant to be platform independent even though it's a `Unit`. You can see each coroutine platform define it in a similar way [here](https://github.com/search?q=repo%3AKotlin%2Fkotlinx.coroutines+public+actual+typealias+TestResult&type=code).

I know the IDE says it's unused, but let's keep it.